website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -112,8 +112,8 @@ api.getGroupTasks = {
 };
 
 /**
- * @api {post} /api/v3/group/:groupId/tasks/:taskId/move/to/:position Move a group task to
- * a specified position
+ * @api {post} /api/v3/group/:groupId/tasks/:taskId/move/to/:position
+ * Move a group task to a specified position
```

Comment:
Interesting! Does breaking that `@api` line at the start of the text instead of in the middle fix the problem of only the second part of the text appearing in the left-hand column of the apidocs page?

If so, that's better than my fix of putting it all on one line and disabling the lint max-length check. :) https://github.com/HabitRPG/habitica/pull/11832/files#diff-754aecadb3f23b61d07ebc934534a697

Reply:
Yes, it does! I hadn't realized there was a max-length check until it tripped an error (I had originally put those all on the first line with the URL), and while thinking of how to resolve it without just getting rid of that change, I wondered about that and tested it out to make sure it worked. I didn't occur to me to disable the lint check (and even if it did I probably would have avoided it XD).